### PR TITLE
Extend OP_RETURN from 40 to 83 for City Chain

### PR DESCRIPTION
**Pull request by sondreb**

### CODE REVIEW

**Comment by dangershony** [comment]
Do we need a new release?

File: src/Directory.Build.props [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <Version>1.0.21</Version>
```
Comment (dangershony) [inline src/Directory.Build.props:3]: Isn't it better to  bump version in its own PR (for next time)

Comment (sondreb) [reply src/Directory.Build.props:3]: Maybe we can do it in separate PRs, but it doesn't change much having it in any of the PRs that comes in between releases. It's just so the draft-release is bumped.

**Comment by sondreb** [comment]
Don't merge this until the City Chain project is ready for the change.

**Pull request merged**